src/nblocks.module.ts

```diff
@@ -13,9 +15,11 @@ import { AuthModule } from './nebulr-auth/nebulr-auth.module';
     UserModule,
     TenantModule,
     FileModule,
-    WebhooksModule
+    WebhooksModule,
+    CacheModule.register(),
   ],
   controllers: [],
-  providers: [],
+  providers: [CacheService, ClientService],
+  exports: [CacheService, ClientService],
```

Comment:
CacheService is not a service that should be used by developer, remove this. But `NebulrAuthService` is missing. Provide and export `NebulrAuthService`.